Makefile

```diff
@@ -19,3 +20,10 @@ lint/helm:
 	# TODO(jawnsy): enable --strict once we fix the warnings
 	helm lint .
 .PHONY: lint/helm
+
+docs: README.md
+.PHONY: docs
+
+README.md: README.md.gotmpl
+	@echo "--- Generating documentation"
+	helm-docs --template-files=$<
```

Comment:
Should we still use prettier to format the table after generation?

Reply:
yeah, will do in a follow-up though, because then I also have to install prettier :sweat: